src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -121,7 +121,7 @@ class NUnit extends ConventionTask {
 
     String getNunitName() {
         if (isV35OrAbove) {
-            return "NUnit.${getNunitVersion()}"
+            return "NUnit.Console-${getNunitVersion()}"
```

Comment:
This is getting super weird. They are not consistent at all :(

https://github.com/nunit/nunit-console/releases

3.6.0 doesn't even have the .zip file, and has different name :(